eng/targets/Settings.props

```diff
@@ -208,8 +208,7 @@
                       and hence suppress this warning until we get closer to release and a more
                       thorough documentation story
         -->
-        <!-- PROTOTYPE(extended-property-patterns) PublicAPIs -->
-        <NoWarn>$(NoWarn);1573;1591;1701;RS0016</NoWarn>
+        <NoWarn>$(NoWarn);1573;1591;1701</NoWarn>
```

Comment:
Thanks a lot for sorting this out. Have we fixed the code action or this is done manually? (I have no idea how this could be done manually)

Reply:
I've done it semi-manually: copy sections of generated code into real files, then trigger publicAPI fixer on those, repeat... It's better than manually editting, by far :-)